webcompat/views.py

```diff
@@ -73,10 +73,15 @@ def format_date(datestring):
 @app.route('/login')
 def login():
     if session.get('user_id', None) is None:
-        # manually set the referer so we know where to come back to
-        # when we return from GitHub
-        set_referer(request)
-        return github.authorize('public_repo')
+        if app.config['TESTING']:
+            session['username'] = 'testuser'
+            session['avatar_url'] = '/img/avatar.png?'
```

Comment:
Ah OK, makes sense. I'm guessing the github package prepends a GET param (and expects to be able to do something useful with it). Seems fine to leave in.